plugins/modules/cargo.py

```diff
@@ -44,6 +44,13 @@
       try to install all of them in this version.
     type: str
     required: false
+  locked:
+    description:
+      - Install with locked dependencies.
```

Comment:
I'm not sure whether anyone could assume this is used for idempotency checks, so I would explicitly state that it isn't:
```suggestion
      - Install with locked dependencies.
      - This is only used when installing packages.
```